front/src/pages/MainPage/components/Grid/index.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import { useDispatch, useSelector } from "react-redux";
+
+import { selectBooks, addItemToBasket } from "features/books/booksSlice";
+import { checkSingleStatus } from "utils/checkSingleStatus";
+
+import { Card } from "./components";
+import { useStyles } from "./styles";
```

Comment:
to jest przestarzała metoda (za to uzywa sie jej np. w React Native), uzyj styled-components, a najlepiej emotion. Ta sama skladnia, ciut wydajniejsze

https://github.com/emotion-js/emotion

Reply:
Done.